evm-contracts/src/v0.6/dev/PrepaidAggregator.sol

```diff
@@ -67,6 +67,7 @@ contract PrepaidAggregator is AggregatorInterface, Owned, WithdrawalInterface {
   LinkTokenInterface private LINK;
   mapping(address => OracleStatus) private oracles;
   mapping(uint32 => Round) internal rounds;
+  mapping(address => bool) internal requesters;
```

Comment:
To match the existing Aggregator, we should name this `authorizedRequesters` and make it public.